code/game/objects/structures/girders.dm

```diff
@@ -376,9 +381,8 @@
 	else
 		..()
 
-/obj/structure/girder/cult/blob_act()

```

Comment:
Now that cult girders are a subtype of girders it's duplicated from girder/blob_act()